dynomitemanager/src/main/java/com/netflix/dynomitemanager/supplier/EurekaHostsSupplier.java

```diff
@@ -15,6 +15,13 @@
  */
 package com.netflix.dynomitemanager.supplier;
 
+import java.util.ArrayList;
```

Comment:
no reason. I use eclipse and run CTRL + SHIFT + O to organized imports... I have revert-it as you like it...